mondrian/src/main/java/mondrian/rolap/SqlContextConstraint.java

```diff
@@ -282,10 +282,8 @@ public void addMemberConstraint(
             evaluator.restore(savepoint);
         }
 
-        // comment out addMemberConstraint here since constraint
-        // is already added by addContextConstraint
-        // SqlConstraintUtils.addMemberConstraint(
-        //        sqlQuery, baseCube, aggStar, parent, true);
+         SqlConstraintUtils.addMemberConstraint(
```

Comment:
Hey @Advent51 - just so I understand, why is the comment from before your change not valid?  I.e. is already added by addContextConstraint is already added by addContextConstraint?